packages/dgt-components/lib/components/authentication/authenticate.services.spec.ts

```diff
@@ -70,6 +72,7 @@ describe('AuthenticateServices', () => {
 
       expect(result.webId).toBe(event.webId);
       expect(result.validationResults).toHaveLength(0);
+      expect((utils as any).addProtocolPrefix).toHaveBeenCalledTimes(1);
```

Comment:
```suggestion
```

Reply:
`addProtocolPrefix` should still always have been called in this case, no?